src/main/java/com/ixortalk/authserver/config/OAuth2ServerConfiguration.java

```diff
@@ -153,9 +144,15 @@ public void cleanUpClientsFromConfiguration() {
             @Inject
             private ManagementServerProperties managementServerProperties;
 
+            @Inject
+            private IxorTalkProperties ixorTalkProperties;
+
             @Override
             protected void configure(HttpSecurity http) throws Exception {
                 // @formatter:off
+                SimpleUrlLogoutSuccessHandler logoutSuccessHandler =  new SimpleUrlLogoutSuccessHandler();
+                logoutSuccessHandler.setDefaultTargetUrl("/"+ixorTalkProperties.getLogout().getDefaultRedirectUri());
```

Comment:
The leading `/` should be part of the config property too, should be passed as-is here.